src/pages/Launchpad/Progress.tsx

```diff
@@ -46,25 +48,25 @@ export default function Progress({ step, classes = "" }: Props) {
 
       {isOtherStepsShown && (
         <>
-          <Step isDone={p >= 2} isCurr={step === 2}>
+          <Step isDone={p >= 2} isCurr={step === 2} step={steps[2]}>
```

Comment:
you could map `<Step/>` by creating an array/object of step titles

also, `isDone` and `isCurr` is better computed inside `<Step/>` now that `step` is passed as props